cs/src/Connections/TunnelRelayTunnelClient.cs

```diff
@@ -116,28 +116,6 @@ protected override Task<ITunnelConnector> CreateTunnelConnectorAsync(Cancellatio
         return Task.FromResult(result);
     }
 
-    /// <summary>
-    /// Connect to the clientRelayUri using accessToken.
-    /// </summary>
-    protected Task ConnectAsync(
```

Comment:
This is used by the Relay service [here](https://devdiv.visualstudio.com/OnlineServices/_git/tunnels?path=/src/Tunnel/Src/DataPlane/Relay/WebForwarding/RelayServiceTunnelClient.cs&version=GBmain&line=127&lineEnd=128&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents). Is there going to be an alternative?

Reply:
It's a mistake, I will restore it.

Reply:
Done.